.github/CODEOWNERS

```diff
@@ -239,7 +239,7 @@
 /comp/collector @DataDog/agent-metrics-logs
 /comp/core @DataDog/agent-shared-components
 /comp/dogstatsd @DataDog/agent-metrics-logs
-/comp/forwarder @DataDog/agent-shared-components
+/comp/forwarder @DataDog/agent-processing-and-routing
```

Comment:
This is automatically generated: https://github.com/DataDog/datadog-agent/pull/24188/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R233

Reply:
Oh interesting, can you point me how it's assigned to teams, please?

Reply:
It is managed by [inv lint-components](https://github.com/DataDog/datadog-agent/pull/24188/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R233) which uses `team` flag (`// team: agent-shared-components`) from component.go files.